src/WorkOS.net/Services/MFA/_interfaces/EnrollFactorResponse.cs

```diff
@@ -0,0 +1,34 @@
+namespace WorkOS
+{
+    using Newtonsoft.Json;
+
+    /// <summary>
+    /// The response from the WorkOS API when enrolling a Factor.
+    /// </summary>
+    public class EnrollFactorResponse
```

Comment:
We'll need to return the values nested under the `totp` field as well if it's a totp factor.

```
{
    "object": "authentication_factor",
    "id": "auth_factor_01FW48AGJ0CYRSY9XYNDA5B4JV",
    "created_at": "2022-02-17T16:30:25.342Z",
    "updated_at": "2022-02-17T16:30:25.342Z",
    "type": "totp",
    "environment_id": "environment_01EPZWK497BAJ96SW5Q99RWH3C",
    "totp": {
        "qr_code": "data:image/png;base64,{really long value}",
        "secret": "IBRRO5BUIREHMGIEN5HUQPSWKF6SWTA3",
        "uri": "otpauth://totp/WorkOS:some_user?secret=IBRRO5BUIREHMGIEN5HUQPSWKF6SWTA3&issuer=WorkOS"
    }
}
```